task04/test/unit/helpers.test.ts

```diff
@@ -0,0 +1,35 @@
+import {validateInput, shortenPublicHoliday} from '../../helpers'
+import {holidays, shortHolidays} from '../test-data'
+import {SUPPORTED_COUNTRIES} from "../../config";
+
+const year = 2024;
+const country = SUPPORTED_COUNTRIES[0];
+
+describe('Validate input', () => {
```

Comment:
A small note, try to use the names of services or endpoints (depending on the type of tests) in the `describe` block for better clarity, if you need more information, you can always use the nested `describe` statement.